c_common/front_end_common_lib/src/recording.c

```diff
@@ -59,7 +59,7 @@ struct recording_data_t {
     uint32_t time_between_triggers;
     uint32_t last_sequence_number;
     recording_channel_t *region_pointers[0];
-};
+} recording_data_t;
```

Comment:
For consistency this should be the same as the struct above it i.e. there's a missing "typedef" here.

Reply:
wonder how i missed that. fixed in last commit. wonder why the compiler was happy. lol

Reply:
I don't think the compiler necessarily cares in this case, but it seemed sensible to suggest doing the same thing for each struct.